esm-samples/jsapi-angular-cli/README.md

```diff
@@ -43,15 +43,15 @@ To use the CSS at a component-level, set a CSS link in `index.html`:
   <link rel="stylesheet" href="assets/esri/themes/dark/main.css">
 ```
 
-**Step 4** Working with Routes - when building routes, set the base url for the `/assets` folder. Development and deployment will require setting different urls. 
+**Step 4** Set the base url for the assets folder so they are correctly resolved. 
```

Comment:
Maybe add a note for when working with routes, saying to use a path that is not relative to the page's path, like `/assets`.

Reply:
Good point. It's probably best to just leave it as `/assets` since that will always work in Angular and wont' trip someone up if they bolt in a route.